src/drivers/storage/storage.h

```diff
@@ -0,0 +1,35 @@
+#ifndef _STORAGE_H_
+#define _STORAGE_H_
+
+#include <inttypes.h>
+
+#define DEFAULT_SSID		"NerdMinerAP"
+#define DEFAULT_WIFIPW		"MineYourCoins"
+#define DEFAULT_POOLURL		"public-pool.io"
+#define DEFAULT_WALLETID	"yourBtcAddress"
+#define DEFAULT_POOLPORT	21496
+#define DEFAULT_TIMEZONE	2
+#define DEFAULT_SAVESTATS	false
+
+// JSON config file
+#define JSON_CONFIG_FILE "/config.json"
+#define JSON_KEY_SSID	"SSID"
+#define JSON_KEY_PASW	"PW"
+#define JSON_KEY_POOLURL	"PoolUrl"
+#define JSON_KEY_WALLETID	"BtcWallet"
+#define JSON_KEY_POOLPORT	"PoolPort"
+#define JSON_KEY_TIMEZONE	"Timezone"
+#define JSON_KEY_STATS2NV	"saveStats"
+
+struct TSettings
+{
+	char WifiSSID[80]{ DEFAULT_SSID };
+	char WifiPW[80]{ DEFAULT_WIFIPW };
+	char PoolAddress[80]{ DEFAULT_POOLURL };
+	char BtcWallet[80]{ DEFAULT_WALLETID };
+	uint32_t PoolPort{ DEFAULT_POOLPORT };
+	uint32_t Timezone{ DEFAULT_TIMEZONE };
```

Comment:
Please consider changing the type to suit timezones less than zero. (-3 Brazil)

Reply:
Oh thanks. How could i miss that? :)

Reply:
this is just for me to remember, jeje.

And again, thanks for your contribution

  json["poolString"] = poolString;
  json["portNumber"] = portNumber;
  json["btcString"] = btcString;
  json["gmtZone"] = GMTzone;
  json["saveStatsToNVS"] = String(saveStatsToNVS);

Reply:
> ok, Elmo, I'll have to make a PR to bring back same used names to not have to reedit all config from active Nerdminers. Even I like this new names, this a real problem for those who update.

Hi.
Sure, np, do whatever is neccessary.